packages/tsconfig-reference/copy/id/sections/top_level.md

```diff
@@ -0,0 +1,3 @@
+### Sumber Bidang
+
+Memulai dari pilihan di dalam sumber TSConfig - banyak pilihan bagaimana berhubungan dengan projek Typescript atau Javascript yang kamu buat.
```

Comment:
Menurut saya `these options relate to how ...` itu kurang lebih `Pilihan-pilihan ini berhubungan dengan bagaimana...`

Kemudian kemarin kita sepakat untuk menggunakan `Anda` daripada `kamu` Mas, diskusinya bisa dilihat di sini
https://github.com/microsoft/TypeScript-Website/pull/963#discussion_r475663161